From b7c28afa17ad65ed3345b2462b5edb364a484a2e Mon Sep 17 00:00:00 2001
From: Edoardo Morandi <edoardo.morandi@m4ss.net>
Date: Thu, 5 Dec 2024 16:11:36 +0100
Subject: [PATCH] remove exe files

This is necessary for the native variant of the package, because they
are already present in the perl-native package.

Signed-off-by: Edoardo Morandi <edoardo.morandi@m4ss.net>
Upstream-Status: Pending
---
 Makefile.PL | 17 +----------------
 1 file changed, 1 insertion(+), 16 deletions(-)

diff --git a/Makefile.PL b/Makefile.PL
index 6f1a674..96895a6 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -25,25 +25,10 @@ WriteMakefile
         'List::Util' => 0,
         'version' => 0.88,
     },
-    'EXE_FILES' => [ _scripts() ],
+    'EXE_FILES' => [],
     'INSTALLDIRS' => ($] < 5.011 ? 'perl' : 'site'),
     'PL_FILES' => {},
     LICENSE => 'perl',
     @extra,
 )
 ;
-
-sub _scripts {
- my $scripts = 'corelist';
- if ( $] >= 5.008009 and !$ENV{PERL_CORE} ) {
-   require Config;
-   my $version = sprintf("%vd",$^V);
-   if ( $Config::Config{versiononly} and
-      $Config::Config{startperl} =~ /\Q$version\E$/ ) {
-      require File::Copy;
-      File::Copy::copy( 'corelist', "corelist$version" );
-      $scripts = "corelist$version";
-    }
- }
- return $scripts;
-}
